### ISSUE TITLE
I'm successfully hacked a zeronet node from another node! Tor did not helped! Check this out! 

### ISSUE THREAD

**Issue by zeronethacked** [zeronethacked/zeronethacked#1]
I have a bad news @imachug @filips123 @shortcutme! I was able to hack ZeroNet and I documented this using a screen recorder. I zipped the video and uploaded to file.io[.](https://github.com/HelloZeroNet/ZeroNet/issues/2273) 

If a link doesn't work try the next one or request me to upload the video to YouTube.

- https://file.io/JBjo9Y
- https://file.io/GUQVfG
- https://file.io/xSxwKi

Greetings,
ZeroNet Hacker

**Comment by imachug** [comment]
ZIP bomb? So clever.

**Comment by filips123** [comment]
[ZIP bomb](https://bugs.python.org/issue36462) should be fixed in recent Python versions. Also you before unzipping files, we can check real size to determine if it looks too much.

@zeronethacked First two links do not work anymore.

**Comment by imachug** [comment]
@filips123 Lol, he actually didn't upload any zip bombs to ZeroNet. The files he linked to were zip bombs themselves.

**Comment by filips123** [comment]
Yes, I know. I just said that it shouldn't be a big problem with ZeroNet either as Python should have some protections for this.

Also, most modern package managers (at least on Windows and Android) will show you actual size of content that is in ZIP file so  you can see that is is ZIP bomb easily.
